makeUniversalBinary.sh

```diff
@@ -38,3 +38,19 @@ for staticLibPath in ${ARM64_LIB}/*.a; do
    fi
 done
 
+find ${ARM64_LIB} -name "*.dylib" -type f -exec chmod +w {} \;
```

Comment:
Was there anything special you had to do differently for dylibs? I'm wondering if we could just change the find invocation to find both `.a` and `.dylib` and eliminate the duplication.